### DISCUSSION TITLE
World of Warcraft custom profile (Elvui/Windtools etc.)

### DISCUSSION THREAD

**Discussion by xWolpertinger01** [catppuccin/catppuccin#2235] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

I played WoW and thought since there are already add-ons which can be used by (officially allowed) to design the user-interface, colors etc. I thought a custom profile with the colors and look of the Catppuccin-theme would be really awesome if possible as it would refresh the usual boring vanilla World of Warcraft-style.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
